free-programming-books-zh.md

```diff
@@ -466,27 +466,27 @@
 * [Nutz-book Nutz烹调向导](http://nutzbook.wendal.net)
 * [Nutz文档](https://nutzam.com/core/nutz_preface.html)
 * [REST 实战](https://github.com/waylau/rest-in-action)
-* [Spring Boot参考指南](https://github.com/qibaoguang/Spring-Boot-Reference-Guide) (翻译中)
+* [Spring Boot参考指南](https://github.com/qibaoguang/Spring-Boot-Reference-Guide) (:编著状态: *翻译中*)
 * [Spring Framework 4.x参考文档](https://github.com/waylau/spring-framework-4-reference)
 * [用jersey构建REST服务](https://github.com/waylau/RestDemo)
 
 
 ### Javascript
 
 * [Airbnb JavaScript 规范](https://github.com/adamlu/javascript-style-guide)
-* [ECMAScript 6 入门](http://es6.ruanyifeng.com) (作者：阮一峰)
+* [ECMAScript 6 入门](http://es6.ruanyifeng.com) - 阮一峰
 * [Google JavaScript 代码风格指南](http://bq69.com/blog/articles/script/868/google-javascript-style-guide.html)
 * [JavaScript Promise迷你书](http://liubin.github.io/promises-book/)
 * [Javascript 原理](https://web.archive.org/web/20170112164945/http://typeof.net/s/jsmech/)
 * [JavaScript 标准参考教程（alpha）](http://javascript.ruanyifeng.com)
-* [《JavaScript 模式》](https://github.com/jayli/javascript-patterns) “JavaScript patterns”中译本
+* [《JavaScript 模式》](https://github.com/jayli/javascript-patterns) (《JavaScript patterns》译本)
 * [javascript 的 12 个怪癖](https://github.com/justjavac/12-javascript-quirks)
 * [JavaScript 秘密花园](http://bonsaiden.github.io/JavaScript-Garden/zh/)
-* [JavaScript核心概念及实践](http://icodeit.org/jsccp/) (PDF) (此书已由人民邮电出版社出版发行，但作者依然免费提供PDF版本，希望开发者们去购买，支持作者)
+* [JavaScript核心概念及实践](http://icodeit.org/jsccp/) (PDF) (此书已出版发行，但作者依然提供免费PDF版本，希望开发者们去购买，支持作者)
```

Comment:
I think maybe we should remove comment too.

Reply:
Agree , will remove it